common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -96,6 +96,7 @@ public class MixAll {
     public static final String REPLY_MESSAGE_FLAG = "reply";
     public static final String LMQ_PREFIX = "%LMQ%";
     public static final String MULTI_DISPATCH_QUEUE_SPLITTER = ",";
+    public static final String PATH_SEPARATOR = "/";
```

Comment:
Should we consider other operating systems, such as Windows, whose path delimiter is \?

Reply:
+1, could use File.separator or System.getProperty("file.separator") @humkum